spec/controllers/openid_connect/authorizations_controller_spec.rb

```diff
@@ -0,0 +1,178 @@
+require "spec_helper"
+
+describe OpenidConnect::AuthorizationsController, type: :controller do
+  let!(:client) { OAuthApplication.create!(name: "Diaspora Test Client", redirect_uris: ["http://localhost:3000/"]) }
+  let!(:client_with_multiple_redirects) { OAuthApplication.create!(name: "Diaspora Test Client", redirect_uris: ["http://localhost:3000/","http://localhost/"]) }
+
+  before do
+    sign_in :user, alice
+    allow(@controller).to receive(:current_user).and_return(alice)
+    Scope.create!(name: "openid")
+  end
+
+  describe "#new" do
+    context "when valid parameters are passed" do
+      render_views
+      context "as GET request" do
+        it "should return a form page" do
+          get :new,
+              {
+                client_id: client.client_id,
+                redirect_uri: "http://localhost:3000/",
+                response_type: "id_token",
+                scope: "openid",
+                nonce: SecureRandom.hex(16),
+                state: SecureRandom.hex(16)
+              }
+          expect(response.body).to match("Diaspora Test Client")
+        end
+      end
+
+      context "as POST request" do
+        it "should return a form page" do
+          post :new,
+              {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.